### PR TITLE
add changes to test_calcCE.R

### PR DESCRIPTION
**Pull request by LeylayqZhang**

### CODE REVIEW

**Review by TwinGan — CHANGES_REQUESTED** [review]
plz check t_range, rh _range, v_range

File: tests/testthat/test_calcCE.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#source("/Users/yiqingzhang/comf/R/calcCE.R")
+library(testthat)
+
+test_that("Testing cooling effect", {
+  # Define the ranges and expected results
+  t_range <- seq(10, 30, 10)
```
Comment (TwinGan) [inline tests/testthat/test_calcCE.R]: please check these three variables

File: tests/testthat/test_calcCE.R [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    expect_equal(
+      round(calcCE(ta, ta, vel, rh, clo = 0.5, met = 1), 2),
+      expected_results[i],
+      tolerance = 0.1
+    )
+  }
+  
+  # Additional specific tests for edge cases
+  expect_equal(calcCE(25, 25, 0.05, 50, clo = 0.6, met = 1), 0)
+  expect_equal(round(calcCE(25, 25, 0.5, 50, clo = 0.6, met = 1), 2), 2.17)
```
Comment (TwinGan) [inline tests/testthat/test_calcCE.R]: why add round in here？

**Review by TwinGan — APPROVED** [review]
[pass]
